frontend/assets/l10n/app_en.json

```diff
@@ -184,5 +184,11 @@
     "showOnMap": "Show on map",
     "unknownCategory": "Unknown category",
     "website": "Website"
+  },
+  "favorites": {
+    "title": "Favorites",
+    "noFavoritesFound": "You have not added any acceptance points to your favorites yet.",
+    "favoriteHasBeenAdded": "The acceptance point has been added to the favorites.",
+    "favoriteHasBeenRemoved": "The acceptance point has been removed from the favorites."
```

Comment:
I used "point" because we use it in other translations:
```
    "noAcceptingStoresFound": "No acceptance points found matching this search.",
    "acceptingStore": "Acceptance points",
    "acceptingStoreNotFound": "Acceptance point not found.",
```
Any objections if I update it everywhere then?

Reply:
ah okay so. then my mistake. I was checking the database name and thought it sounds more intuitive to me. But you can also keep it as it is then.